packages/components/src/internal/components/entities/actions.ts

```diff
@@ -112,6 +141,58 @@ export async function getOperationConfirmationData(
     });
 }
 
+export function getContainersForPermission(permission: PermissionTypes): Promise<string[]> {
+    return new Promise((resolve, reject) => {
+        Security.getContainers({
+            containerPath: getProjectPath(),
+            includeStandardProperties: false,
+            includeSubfolders: true,
```

Comment:
I think we have similar calls to this, but this is a good one that should get some perf testing with a large number of app subfolders.

Reply:
Yes, it was generalized from what we had been using for cross-folder operations for inventory. Good call about perf testing here as well.